docker/Dockerfile

```diff
@@ -37,7 +38,8 @@ RUN apk --no-cache add --virtual build-dependencies \
                   ttf-ubuntu-font-family \
                   libreoffice \
                   redis \
-                  postgresql-client
+                  postgresql-client \
+                  shared-mime-info
```

Comment:
I think that `shared-mime-info` is already installed but it doesn't hurt to list it here, especially as it is now a dependency.